package.json

```diff
@@ -106,7 +106,7 @@
     "react-native-keyboard-aware-scroll-view": "^0.8.0",
     "react-native-modal": "^6.5.0",
     "react-native-recyclerview-list": "git+https://github.com/wordpress-mobile/react-native-recyclerview-list.git#v1.0.0",
-    "react-native-svg": "git+https://github.com/wordpress-mobile/react-native-svg.git#2b76859a986d35d741c9160aaad54aa46f268c6e",
+    "react-native-svg": "git+https://github.com/wordpress-mobile/react-native-svg.git#6ac5592aca612a893dabfa6a618736e552c4f67b",
```

Comment:
Friendly reminder to update the hash now that [the react-native-svg side PR](https://github.com/wordpress-mobile/react-native-svg/pull/1) has been merged. 🙇‍♂️